docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1214,6 +1222,21 @@ In other words the extension will use whatever cluster `kubectl` uses. The same
 
 At the moment no additional options are provided for further customization.
 
+=== Remote Debugging
+
+To remotely debug applications that are running on a cloud environment, we need to deploy the application as described in the previous section and additionally appending the property `quarkus.kubernetes.remote-debug.enabled=true`. This property will automatically configure the Java application to append the java agent configuration (for example: `-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005`) and also the service resource to listen the java agent port. 
+
+After your application has been deployed with the debug enabled, next you need to port-forward the service java agent port:app-name:
```

Comment:
`... you need to to port-forward the service java agent port:app-name:` --> `... you need to tunnel the traffic from your local host machine to the specified port of the java agent`

Reply:
changed